src/builtin.jq

```diff
@@ -265,7 +265,8 @@ def pick(pathexps):
   . as $in
   | reduce path(pathexps) as $a (null;
       setpath($a; $in|getpath($a)) );
-
+# ensure the output of debug(m1,m2) is kept together:
+def debug(msgs): (msgs | debug | empty), .;
 # SQL-ish operators here:
```

Comment:
add blank lines to separate it from `pick/1` and SQL-like functions?

Reply:
Done.